README.md

```diff
@@ -64,4 +64,12 @@ gem install bundler wdm tzinfo-data
 gem update listen middleman
 ```
 
-Once Middleman comes up, you'll be prompted by Windows Firewall. Click "Allow access" and you'll be in business!
+**NOTE**: if you get an error like this:
```

Comment:
Excellent feedback re: "NOTE"s- I totally agree, even if I do have a bad habit of making them :smile: I try to use them as placeholder for needing to cleanup, which didn't happen. Will clean these occurrences up!